diskv.go

```diff
@@ -403,6 +433,9 @@ func (d *Diskv) EraseAll() error {
 	defer d.mu.Unlock()
 	d.cache = make(map[string][]byte)
 	d.cacheSize = 0
+	if d.TempDir != "" {
+		os.RemoveAll(d.TempDir) // errors ignored
```

Comment:
is there a reason other than cleanliness that we need to remove this dir completely? this means that identically-configured cross-process instances will stomp on each other, which defeats one of the benefits of using atomic operations. also, it might be really surprising to someone that set TempDir to `$TMP`